src/matlab2tikz.m

```diff
@@ -2026,6 +2044,27 @@ function assertRegularAxes(handle)
                                        markerInfo.options);
         end
     end
+
+    type = getOrDefault(h, 'Type', 'none');
+    %If the current line is of type errorbar
+    if strcmp(type, 'errorbar')
+        %Get the 'capSize' size of the errorbar marker and the line width of the
+        %errorbar line.
+        capSize = get(h, 'CapSize');
+        lineWidth = get(h, 'LineWidth');
+        %Append the 'error bar style' parameter, which determines the style
+        %of the vertical error bar line, to drawOptions
+        drawOptions = opts_add(drawOptions, ...
+            'error bar style', ...
+            sprintf('{line width=%.1fpt}', lineWidth));
+
+        %Append the 'error mark options' parameter, which determines the
+        %style of the error bar 'marker', to drawOptions
+        drawOptions = opts_add(drawOptions, ...
+            'error mark options', ...
+            sprintf('{\nrotate=90,\nmark size=%.1fpt,\nline width=%.1fpt\n}',...
```

Comment:
It would be better to use the `opts_...`  functions (especially `opts_addSubOpts`) and `formatDim` (if I remember correctly) to ensure uniform formatting of those option arrays.
In the future, I expect that we will provide options to customize these formatting options to your liking, but for that to happen, we all need to use those functions and get rid of string literals as much as possible.